parser.py

```diff
@@ -392,19 +319,19 @@ def mixedlist(self, *args):
         else:
             # <id> [<nninteger>]
             wires = [format_wires(args)]
-        return NamedList('mixedlist', wires)
+        return NamedList("mixedlist", wires)
 
     def argument(self, *args):
         args = unpack(args)
         # <id> or
         # <id>[<nninteger>]
-        return NamedList('argument', args)
+        return NamedList("argument", args)
 
-    def explist(self, *args):
+    def expressionlist(self, *args):
         flat_list = flatten_recursive_list(*args)
-        return NamedList('explist', flat_list)
+        return NamedList("expressionlist", flat_list)
 
-    def exp(self, *args):
+    def expression(self, *args):
```

Comment:
I was getting issues; the name of this rule was clashing with the alias of the `exp` unary op.

Reply:
:thinking: not sure why you were getting an error. The `exp` unary op was given the alias `"exponentiate"` in the grammar

Reply:
Note that all the names are inherited from the original spec, except for the new ones that I explicitly added

Reply:
Nevermind, I see now where the name is doubly-defined in python

Reply:
Rather than rename the terms in the grammar, I think it's more friendly to create the two aliases `"unary_exp"` and `"binary_exp"`